JSDemos/Demos/TagBox/TagCountLimitation/Angular/app/app.component.ts

```diff
@@ -19,9 +19,11 @@ if(!/localhost/.test(document.location.host)) {
 
 export class AppComponent {
     products: Product[];
+    slicedProducts: Product[];
 
     constructor(service: Service) {
         this.products = service.getProducts();
+        this.slicedProducts = service.getProducts().slice(0, 5);
```

Comment:
suggestion:
if replace this to
this.slicedProducts =  this.products.slice(0, 5);
you reduce service calls count.

Reply:
yep, ty